client/lib/css-hot-reload/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import io from 'socket.io-client';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'i18n-calypso';
+
+// https://developer.mozilla.org/en/docs/Web/HTML/Element/link
+let standardAttributes = [
+	'crossorigin',
+	'href',
+	'hreflang',
+	'media',
+	'rel',
+	'sizes',
+	'title',
+	'type'
+];
+
+/**
+ * Replaces the old hash parameter with the current timestamp if `name` is `href`.
+ * @return {Object}
+ */
+function bustHashForHrefs( { name, oldValue } ) {
+	// http://some.site.com/and/a/path?with=a&query -> http://some.site.com/and/a/path?v=13508135781
+	const value = 'href' === name
+		? `${ oldValue.split( '?' ).shift() }?v=${ new Date().getTime() }`
+		: oldValue;
+
+	return { name, value };
+};
+
+/**
+ * @return {Boolean}
+ */
+function isChanged( href, changedFiles ) {
+	// "/calypso/style-debug.css?v=5a1db7fee7" -> "style-debug.css"
+	const path = href.split( '?' ).shift().split( '/' ).pop();
+
+	return changedFiles.some( file => file === path );
```

Comment:
might as well use the **lodash** utility here. not a big deal either way though…

```js
return changedFiles.some( matches( path ) );
```